src/patterns/error-validation/examples/errors-proto/errors/index.njk

```diff
@@ -299,6 +308,30 @@ layout: none
                 }
             })
         }}
+
+        {{
+            onsDuration({
+                "label": "Duration",
+                "id": "address-duration",
+                "dontWrap": true,
+                "error": {
+                    "text": "Enter a number that is less that 60",
+                    "id": "duration-error"
+                },
+                "field1": {
+                    "id": "address-duration-hours",
+                    "name": "address-duration-hours",
+                    "label": "Hours"
```

Comment:
I take it back, sorry.  We do need a valid value in here (say '2'), because you'd only show this error message if hours was valid but minutes was not.  If they were both invalid then you'd show the "Enter a duration" error first.